rubin_sim/selfcal/generate_catalog.py

```diff
@@ -37,7 +37,8 @@ def build_tree(ra, dec, leafsize=100):
     """Build KD tree on RA/dec and set radius (via setRad) for matching.
 
     ra, dec = RA and Dec values (in radians).
-    leafsize = the number of Ra/Dec pointings in each leaf node."""
+    leafsize = the number of Ra/Dec pointings in each leaf node.
```

Comment:
Maybe there should be a proper Parameters sections in this and other functions in this file.